Tribler/Test/test_i2i_download.py

```diff
@@ -22,13 +23,16 @@ def download_object_ready():
                 UBUNTU_1504_INFOHASH), item_shown_in_list, 'no download in librarylist')
 
         def do_test_i2i():
+            random_port = random.randint(15000, 20000)
```

Comment:
There's a function to obtain a free port. Use that instead of a random one or we will have collisions every now and then